src/core/messaging/handlers/notification.ts

```diff
@@ -69,7 +80,8 @@ const handleNotification = (
             if (requestId) {
                 store.dispatch(openTransactionRequest({ requestId }));
             } else {
-                //
+                // maybe find a way to handle this
```

Comment:
at least send something in sentry for us to be aware that this happens sometimes